src/com/ableton/PythonPackage.groovy

```diff
@@ -0,0 +1,67 @@
+package com.ableton
+
+
+/**
+ * Class to interact with local Python packages.
+ */
+class PythonPackage implements Serializable {
+  /**
+   * Script context.
+   * <strong>Required value, may not be null!</strong>
+   */
+  @SuppressWarnings('FieldTypeRequired')
+  def script = null
+
+  /**
+   * Start of the line setting the Python package version.
+   */
+  String defaultVersionLineBeginning = "__version__ = '"
+
+  /**
+   * End of the line setting the Python package version.
+   */
+  String defaultVersionLineEnding = "'"
+
+  /**
+   * Read a Python file setting the version of a package (usually {@code __init__.py})
+   * and return its version as a {@code String}.
+   * @param args Map of arguments. Valid arguments include:
+   *             <ul>
+   *               </li>
+   *                 {@code filename} (required): The file to read and parse the version
+   *                 out of.
+   *               </li>
+   *               <li>
+   *                 {@code versionLineBeginning} (optional): Start of the line setting
+   *                 the Python package version.
+   *               </li>
+   *               <li>
+   *                 {@code versionLineEnding} (optional): End of the line setting the
+   *                 Python package version.
+   *               </li>
+   *             </ul>
+   * @return The extracted Python package version.
+   */
+  String readVersion(Map args) {
+    assert script
+    assert args.filename
+    String versionLineBeginning = args.versionLineBeginning ?: defaultVersionLineBeginning
+    String versionLineEnding = args.versionLineEnding ?: defaultVersionLineEnding
+
+    String fileContents = script.readFile(args.filename).trim()
+
+    String version
+    fileContents.split('\n').each { line ->
+      if (line.startsWith(versionLineBeginning)) {
+        // Extract the version string from the quotes
+        version = line.replace(versionLineBeginning, '').replace(versionLineEnding, '')
```

Comment:
Shouldn't we `break` once we have what we want?

Reply:
https://stackoverflow.com/questions/3049790/can-you-break-from-a-groovy-each-closure

Reply:
Let's use `find` then. We are actually looking for a particular line.

Reply:
Addressed in 530b7ae.